examples/space/simple_rn.py

```diff
@@ -38,7 +29,7 @@ class SimpleRn(odl.space.base_ntuples.FnBase):
     """The real space R^n, non-optimized implmentation."""
 
     def __init__(self, size):
-        super().__init__(size, np.float)
+        odl.space.base_ntuples.FnBase.__init__(self, size, np.float)
```

Comment:
Perhaps here a `from odl.space.base_ntuples import FnBase` would be more readable.